Source/JavaScriptCore/inspector/remote/RemoteConnectionToTarget.cpp

```diff
@@ -93,7 +96,8 @@ void RemoteConnectionToTarget::close()
         if (RefPtr target = m_target.get()) {
             targetIdentifier = target->targetIdentifier();
 
-            if (m_connected)
+            auto currState = m_connectionState.exchange(ConnectionState::Closed);
+            if (currState == ConnectionState::Connected)
```

Comment:
id inline this as
```
if (std::exchange(m_connectionState, ConnectionState::Closed) == ConnectionState::Connected)
```

Reply:
I can't seem to pass an `atomic` into `std::exchange`.

I'll inline it though.